docs/refcard.adoc

```diff
@@ -35,3 +35,41 @@ a|`smart.testing.apply.to`
 a|`surefire`, `failsafe`
 a|`surefire, failsafe`
 |===
+
+=== Strategies
```

Comment:
Can we give change this? I believe following table has more information than strategies